cursorless-talon/src/cursorless.py

```diff
@@ -16,8 +23,67 @@ def private_cursorless_show_settings_in_ide():
     def private_cursorless_show_sidebar():
         """Show Cursorless-specific settings in ide"""
 
+    def private_cursorless_notify_docs_opened():
+        """Notify the ide that the docs were opened in case the tutorial is waiting for that event"""
+        ...
```

Comment:
What's up with `...`

Reply:
I believe this is the recommended way to create a placeholder action

Reply:
It is?

Reply:
Huh I thought I saw that somewhere but a quick grep through community reveals it's not used at all there. Removed 🤷